Browse Rail: Put browse rail section at top of main instead of bottom

msagolj opened the pull request:
Adding the browse rail `div` at the bottom of `main` causes a Cumulative Layout Shift penalty dropping the score from ~100 to ~75. Moving to top of `main` so score goes back to ~100:

Before:
![image](https://github.com/adobe-experience-league/exlm/assets/37147400/9c18d9ed-c649-4baa-97f6-3c1dd57a6f57)

After:
![image](https://github.com/adobe-experience-league/exlm/assets/37147400/3305dd87-77a6-459e-a822-c792dffba927)

Test URLs:

- Before: https://main--exlm--adobe-experience-league.hlx.live/en/browse/acrobat
- After: https://fix-rail-performance--exlm--adobe-experience-league.hlx.live/en/browse/acrobat

Buuhuu reviewed (changes requested):
this breaks the waitForLcp logic as the first block on the page is not marque anymore.

Instead move the body main style to the styles.css.

msagolj commented:
> this breaks the waitForLcp logic as the first block on the page is not marque anymore.
> 
> Instead move the body main style to the styles.css.

@Buuhuu thx for valuable feedback. Your right , it breaks the lcp block logic. Reverted to adding rail at the end and moved the css for the rail into the main .

Both desktop and mobile (which was not affected anyway) are now at lighthouse 100.
